civihr_employee_portal/views/views_export/views_emergency_contacts.inc

```diff
@@ -121,11 +238,53 @@ $handler->display->display_options['filters']['combine']['fields'] = array(
 
 /* Display: Non Dependant Emergency Contacts */
 $handler = $view->new_display('block', 'Non Dependant Emergency Contacts', 'non_dependant_emergency_contact');
+$handler->display->display_options['defaults']['title'] = FALSE;
+$handler->display->display_options['title'] = 'Emergency contacts';
 $handler->display->display_options['display_description'] = 'A list of emergency contacts where the "dependant" flag is false';
+$handler->display->display_options['defaults']['footer'] = FALSE;
+/* Footer: Global: Text area */
+$handler->display->display_options['footer']['area']['id'] = 'area';
+$handler->display->display_options['footer']['area']['table'] = 'views';
+$handler->display->display_options['footer']['area']['field'] = 'area';
+$handler->display->display_options['footer']['area']['empty'] = TRUE;
+$handler->display->display_options['footer']['area']['content'] = '<a href="/create-emergency-contact/js/view" class="ctools-use-modal ctools-modal-civihr-custom-style  btn btn-primary">
+  <i class="fa fa-user-plus" aria-hidden="true"></i> Add Emergency Contact
+</a>
+';
+$handler->display->display_options['footer']['area']['format'] = 'html_without_filters';
+$handler->display->display_options['defaults']['empty'] = FALSE;
+/* No results behavior: Global: Text area */
+$handler->display->display_options['empty']['area']['id'] = 'area';
+$handler->display->display_options['empty']['area']['table'] = 'views';
+$handler->display->display_options['empty']['area']['field'] = 'area';
+$handler->display->display_options['empty']['area']['empty'] = TRUE;
+$handler->display->display_options['empty']['area']['content'] = 'No dependants listed!';
```

Comment:
I think this should be in the display below that has the "Dependants" title, and "'No emergency contacts listed!" should be here

Reply:
Fixed.